lib/index.js

```diff
@@ -1,5 +1,5 @@
 var AWS = require('aws-sdk'),
-  converter = require('./converter'),
+  s3pagableStream = require('./s3-pagable-stream'),
```

Comment:
It should be spelled "pageable", right? 🤔

Reply:
Let's call it paging-stream, since that's more like what it does.